tensorflow_lite_support/ios/task/vision/sources/TFLObjectDetector.m

```diff
@@ -62,8 +63,16 @@ - (instancetype)initWithObjectDetector:(TfLiteObjectDetector *)objectDetector {
   return self;
 }
 
-+ (nullable instancetype)objectDetectorWithOptions:(nonnull TFLObjectDetectorOptions *)options
++ (nullable instancetype)objectDetectorWithOptions:(TFLObjectDetectorOptions *)options
                                              error:(NSError **)error {
+
+  if (!options) {
```

Comment:
I have added a null check for options here. Please see my comments [here](https://github.com/tensorflow/tflite-support/pull/787/files#r849323481) and LMK if this is needed win practice.

Reply:
Yes that makes sense.